fvm/evm/types/events.go

```diff
@@ -92,13 +92,13 @@ func (p *transactionEvent) ToCadence(location common.Location) (cadence.Event, e
 			cadence.NewField("type", cadence.UInt8Type),
 			cadence.NewField("payload", cadence.StringType),
 			cadence.NewField("errorCode", cadence.UInt16Type),
+			cadence.NewField("errorMessage", cadence.StringType),
 			cadence.NewField("gasConsumed", cadence.UInt64Type),
 			cadence.NewField("contractAddress", cadence.StringType),
 			cadence.NewField("logs", cadence.StringType),
 			cadence.NewField("blockHeight", cadence.UInt64Type),
 			// todo we can remove hash and just reference block by height (evm-gateway dependency)
 			cadence.NewField("blockHash", cadence.StringType),
-			cadence.NewField("errorMessage", cadence.StringType),
 			cadence.NewField("returnedData", cadence.StringType),
```

Comment:
This also doesn't seem to be in the event declaration, same below

Reply:
good catch, updated at [341b455](https://github.com/onflow/flow-go/pull/6089/commits/341b455ede9355f5a8d7b6ef1015a702983e3317)